cli_components.py

```diff
@@ -24,14 +23,26 @@ def welcome_banner():
               f"\n\n(To exit enter 'q')")
 
     @staticmethod
-    def user_main_menu():
+    def display_main_menu():
         """Prints the main user menu."""
         print("Please select from the following options:\n\n"
               "    (1) plan journey\n"
               "    (2) view user stats\n"
               "    (3) register a vehicle\n"
               "    (4) log out\n")
 
+    @staticmethod
```

Comment:
Function that prints out user stats. Simple for now but we can make better later.